test/tracer_logger.spec.js

```diff
@@ -6,7 +6,7 @@
 /* eslint-disable no-unused-expressions */
 'use strict'
 
-const { expect } = require('chai')
```

Comment:
Keeping it explicit is actually quite nice in my opinion

Reply:
reverted in https://github.com/DataDog/dd-wasm-js-rewriter/pull/122/commits/e11a03ebac914e2b78cf0a397f5ba65d40d99274